freebsd.c

```diff
@@ -25,7 +25,7 @@
 #include <sys/types.h>
 #include <net/if.h>
 #include <sys/stat.h>
-#include <sys/disklabel.h>
+#include <sys/disk.h>
```

Comment:
Hi!  Do you know of any older versions of FreeBSD where this change will cause build failures?

Thanks much.